wai-aria/tools/atta-example.py

```diff
@@ -0,0 +1,289 @@
+# atta-example
```

Comment:
If we remove this and the other example, we should also remove the WAIFAKE stub from all the generated tests.  Which is fine by me.  Work for you?